frontend/src/app/coworking/widgets/room-reservation-table/room-reservation-table.widget.html

```diff
@@ -26,4 +38,11 @@
     [disabled]="!selectButton">
     Reserve
   </button>
+
+  <button
+    mat-stroked-button
+    class="button"
+    (click)="openMemberSelectionDialog()">
+    Add Users
+  </button>
```

Comment:
Looping @KrisJordan in - I might argue that moving the selector dropdown right to this page directly might be better from a UI flow perspective, esp on mobile.

For UI purposes, it is also possible to to add a permanent chip for the current user into the selector. So, we can ultimately just one selector field with a permanent "current user" chip, then the ability to add more members. This selector can appear once a room has been selected on the table above (we can do this with `*ngIf`)

If you have questions, feel free to reply here!

Reply:
Left as is for now since Kris has not answered yet!